openage/convert/service/export/png/binpack.pxd

```diff
@@ -35,6 +39,7 @@ cdef class BinaryTreePacker(Packer):
     cdef packer_node *root
 
     cdef void fit(self, block)
+    cdef tuple get_packer_settings(self)
     cdef packer_node *find_node(self, packer_node *root, unsigned int width, unsigned int height)
```

Comment:
`find_node`, `split_node`, `grow_node`, `grow_right`, `grow_left` can now be declared `noexcept` since we cythonized everything in them.